plugins/core/hu.bme.mit.gamma.expression.model/src/hu/bme/mit/gamma/expression/util/ExpressionModelValidator.java

```diff
@@ -651,4 +668,75 @@ else if (variableDeclarationType instanceof TypeReference &&
 		return validationResultMessages;
 	}
 	
+//////////////////////////////////////////////////////////////////////
+	public Collection<ValidationResultMessage> checkArrayTypeDefinition(ArrayTypeDefinition arrayType){
+		Collection<ValidationResultMessage> validationResultMessages = new ArrayList<ValidationResultMessage>();
+		// Array init size must greater then 0
+		if(expressionEvaluator.evaluateInteger(arrayType.getSize()) <= 0) {
```

Comment:
Here and everywhere else: I believe the `evaluateInteger` and `evaluateBoolean` method throws an exception if the given expression is not evaluable (there is a variable reference in there). How is this exception handled?